### PR TITLE
Feature/us5001 canelsummercamp

### PR DESCRIPTION
**Pull request by lknair2015**

### CODE REVIEW

File: Gateway/crds-angular/Services/CampService.cs [hunk 1]
```diff
@@ -354,7 +354,8 @@ public List<MyCampDTO> GetMyCampInfo(string token)
                                 CampName = camp.EventTitle,
                                 CampStartDate = camp.EventStartDate,
                                 CampEndDate = camp.EventEndDate,
-                                EventId = camp.EventId
+                                EventId = camp.EventId,
+                                CampPrimaryContactEmail = _eventRepository.GetEvent(camp.EventId).PrimaryContact.EmailAddress
```
Comment (silbermm) [inline Gateway/crds-angular/Services/CampService.cs]: this should have broken a test. If it didn't that means there was no test for it, can you add tests for this method?

File: crossroads.net/app/camps/dashboard/camps_dashboard.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
       <div class="row is-flex flex-wrap">
 
         <div class="col-md-6 col-sm-12 is-flex flex-col" ng-repeat="data in dashboard.data">
-          <camp-card attendee="dashboard.fullName(data.lastName,data.nickName)" start-date="data.startDate" end-date="data.endDate" camp-title="data.eventTitle" camp-id="data.eventId" camper-id="data.contactId"> </camp-card>
+          <camp-card attendee="dashboard.fullName(data.lastName,data.nickName)" start-date="data.startDate" end-date="data.endDate" camp-title="data.eventTitle" camp-id="data.eventId" camper-id="data.contactId" camp-primary-contact="data.eventPrimaryContact"> </camp-card>
```
Comment (silbermm) [inline crossroads.net/app/camps/dashboard/camps_dashboard.html]: test should be updated to reflect change that camp-card now takes a `camp-primary-contact`

File: Gateway/MinistryPlatform.Translation/Repositories/EventParticipantRepository.cs [hunk 3]
```diff
@@ -150,7 +150,7 @@ public int GetEventParticipantByContactId(int eventId, int contactId)
 
         public DateTime? EventParticipantSignupDate(int contactId, int eventId, string apiToken)
         {
-            var filter = $"Event_ID_Table.[Event_ID] = {eventId} AND Participant_ID_Table_Contact_ID_Table.[Contact_ID] = {contactId}";
+            var filter = $"Event_ID_Table.[Event_ID] = {eventId} AND Participant_ID_Table_Contact_ID_Table.[Contact_ID] = {contactId} AND Participation_Status_ID_Table.[Participation_Status_ID] <> 5";
```
Comment (silbermm) [inline Gateway/MinistryPlatform.Translation/Repositories/EventParticipantRepository.cs]: 5 is a 'magic' number. Should be a config variable.

File: crossroads.net/app/camps/dashboard/camps_dashboard.html (hunk 2 above)
Comment (lknair2015) [inline crossroads.net/app/camps/dashboard/camps_dashboard.html]: updated the test

**Pull request merged**